apps/src/lib/util/AnalyticsReporter.js

```diff
@@ -0,0 +1,134 @@
+import {
+  init,
+  track,
+  Identify,
+  identify
+  // setUserId
+} from '@amplitude/analytics-browser';
+import {currentLocation} from '@cdo/apps/utils';
+
+// A flag that can be toggled to send events regardless of environment
+const ALWAYS_PUT = false;
```

Comment:
I'm not sure this name clearly communicates the intent- maybe something like SEND_OVERRIDE? Or is that not the idea?

Reply:
I like what you updated it to!